### PR TITLE
Plex MultiEpisode Patch (Regex test for file Names)

### PR DESCRIPTION
**Pull request by randomevents**
Using SlrG's tip involving a regex test, replaced my old method (that didn't fully work) that tried to do a courtesy check to make sure the file name was "incompatible" with Plex. Also cleaned up the option description.

### CODE REVIEW

**Comment by SlrG** [comment]
A little typo in the import line. Also on my testing folder on a plex named show, episode thumbs and summary seems to be broken. I'll check a bit more when I get the time. :) After the weekend probably.

**Comment by randomevents** [comment]
import line? I can't see it

File: Contents/Code/__init__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # Cleanup and some extensions by SlrG
 # Logo by CrazyRabbit
 #
-import os, re, time, datetime, platform, traceback, glob, re, htmlentitydefs
+import os, re, time, datetime, platform, traceback, glob, re, htmlentitydefsp
```
Comment (SlrG) [inline Contents/Code/__init__.py:10]: It's only htmlentitydefs not htmlentitydefsp. :)

**Comment by SlrG** [comment]
I think I found what broke the episode thumbs, when your option was enabled. Summaries were due to an error in my nfo files used for testing. Tomorrow I should have a version working, which incorporates your additions. Thank you very much for your contribution. :+1: :)

**Comment by randomevents** [comment]
Oh! I am so sorry. I know exactly why that happened, I would hit control F to search, start typing in "plexpatch" and realize that it was searching from chrome and not from the github div. I didn't see it on mine because I copied the last changes over by hand in my "production" version.

**Pull request merged**